service/collector/elb.go

```diff
@@ -52,6 +52,12 @@ type ELB struct {
 	installationName string
 }
 
+type loadBalancer struct {
+	instancesOutOfService float64
+	name                  string
+	tags                  map[string]string
```

Comment:
I personally prefer to keep structs "ready to be exported". In this case all The fields should be capitalized. Then it's easier to reason about the struct within package.

Reply:
Changed.